index.html

```diff
@@ -215,231 +215,186 @@ <h3>Motivation for a Profile</h3>
-        Users of devices want to process data from all devices that conform to a class of devices in a uniform way.
-        They need a guarantee that they are able to correctly interact with all affordances of the Thing
-        that complies with this class of devices.
-        Different interpretations of the same Thing Description, that lead to different behaviour, should not be
-        possible.
-        Users want to integrate devices in existing scenarios out of the box, i.e. with close to zero configuration
-        tasks.
-      </section>
-    </section>
+    <p>
+      The Thing Description specification is designed to be protocol-agnostic
+      and flexible enough to describe a wide range of existing ("brownfield")
+      IoT devices, rather than specifying a fixed protocol or application
+      programming interface (API) which all devices must implement. The downside
+      of this open ended flexibility is that it makes ad-hoc interoperability on
+      the Web of Things very difficult, because it's nearly impossible to create
+      a single WoT <a href="#dfn-consumer">Consumer</a> [[wot-architecture]]
+      which is guaranteed to be able to communicate with any
+      <a href="#dfn-thing">Web Thing</a> [[wot-architecture]] out of the box.
+    </p>
+    <p>
+      This specification is designed to complement the Thing Description
+      [[wot-thing-description]] specification, by defining a
+      <a href="#profiling-mechanism">Profiling Mechanism</a> and a set of
+      <a href="#dfn-profile">Profiles</a> which enable
+      <a href="#out-of-the-box-interoperability">out-of-the-box
+      interoperability</a> between <a href="#dfn-thing">Web Things</a> and their
+      <a href="#dfn-consumer">Consumers</a>.
+    </p>
+    <p>
+      Profiles are designed specifically for new ("greenfield") implementations
+      where developers have the freedom to conform to a prescriptive protocol
+      binding and set of common constraints, in order to benefit from
+      out-of-the-box interoperability.
+    </p>
+    <p class="note">
+      Of the Profiles defined in this specification, only the
+      <a href="#http-basic-profile">HTTP Basic Profile</a> is currently
+      normative. It is planned that future versions of this specification will
+      normatively define the <a href=#sec-http-sse-profile>HTTP SSE Profile</a>
+      and <a href="#sec-http-webhook-profile">HTTP Webhook Profile</a>, but
+      in this version of the specification they are only informative.
+    </p>
+    <p>
+      Future versions of this specification, or extension specifications, may
+      define additional Profiles.
+    </p>
+  </section>
+  <section id="profile-example">
+    <h2>Profile Example</h2>
+    <p>
+    <a href="#simple-profile-td-sample">Example 1</a> shows a <a>Thing Description</a>
+    instance that follows the <a>HTTP Basic Profile</a> as described in this
+    sepecification.
```

Comment:
I would suggest describing this as "a Thing Description of a Web Thing which follows the HTTP Basic Profile", since it's really the Web Thing that follows the profile, not just the Thing Description.

Reply:
sepecification -> specification